app/services/ip_stack_service_provider.rb

```diff
@@ -0,0 +1,24 @@
+class IpStackServiceProvider < GeolocatorServiceProvider
+  include HTTParty
+
+  base_uri 'api.ipstack.com'
+
+  def initialize(access_credentials)
+    super(access_credentials)
+  end
+
+  def fetch(ip_address)
+    response = self.class.get("/#{ip_address}?access_key=#{@access_credentials}&&hostname=1")
```

Comment:
hostname=1 will be removed on next PR. I got confused with the concept of URL asked in the exercise